lib/Documentable/Index.pm6

```diff
@@ -13,18 +13,17 @@ method new(
     :$meta!,
     :$origin!
 ) {
-
-    my $name;
-    if $meta.elems > 1 {
-        my $last = textify-pod $meta[*-1];
-        my $rest = $meta[0..*-2];
-        $name = "$last ($rest)";
+    my ($name, $category);
+    if $meta.elems == 2 {
+        $category = $meta[0];
+        $name = $meta[1];
     } else {
-        $name = textify-pod $meta;
+        warn "At $origin.url() $meta.raku() is not formatted properly";
```

Comment:
It should probably say what's wrong about the format. Also, is this tested?

Reply:
Added a note, thanks! Plus a test is added.